_health-care/_js/_components/continue-button.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+/**
+ * A component for the continue button to navigate through the sections of questions.
+ */
+class ContinueButton extends React.Component {
```

Comment:
Here...maybe take a `nextUrl` prop that you connect button to. If undefined, then disable the button.